src/routes/user.ts

```diff
@@ -0,0 +1,16 @@
+import { ping, RegisterUser, LoginUser } from "../controllers/user";
+import { Router } from "express";
+import { body } from "express-validator";
+import multer from "multer";
+
+const UserRouter = Router();
+
+const upload = multer({ dest: "./src/images/" });
```

Comment:
this line saving the data to image folder first and then we move it to Cloudinary. This is double cost for the deploy server.

Reply:
We need to move the req object to cloudinary